batch/scripts/unify_projection.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+
+# requires arguments
+# -s | --source
+# -T | --target
+#      --target_crs
+
+ME=$(basename "$0")
+. get_arguments.sh "$@"
+
+echo "Reproject to a common CRS"
+
+src_count=0
+CMD_ARGS=()
+
+for s in ${SRC[@]}; do
+  source_dir="SRC_${src_count}"
+  mkdir -p "$source_dir"
+
+  echo "Now recursively downloading $s to $source_dir"
+  if [[ $s == gs://* ]]; then
+    gsutil cp -m -r "$s" "$source_dir"
+  elif [[ $s == s3://* ]]; then
+    aws s3 cp --no-progress "$s" "$source_dir"
```

Comment:
I know we're not using aws for the source_uri, but shouldn't this have "--recursive" on it to copy an entire folder path and sub-folders?

Reply:
Good catch, thanks!